src/installer.ts

```diff
@@ -40,7 +40,9 @@ export async function installer(): Promise<void> {
   core.exportVariable("OPAMERRLOGLEN", 0);
   core.exportVariable("OPAMJOBS", numberOfProcessors);
   core.exportVariable("OPAMPRECISETRACKING", 1);
-  core.exportVariable("OPAMSOLVERTIMEOUT", 500);
+  // [todo] remove this line once we unlock opam 2.2
```

Comment:
We can live in hope, but opam 2.2 is not guaranteed to fix this! opam 2.1's CUDF trimming has in many (but not all) cases reduced this problem.

Reply:
It's true that 2.1 reduced this problem and I believe it's enough to get rid of it. I'm writing this comment because 2.2 brings native Windows support so we can jump over 2.1 and upgrade to 2.2 on all platforms!